build-system/server/new-server/transforms/transform.ts

```diff
@@ -19,15 +19,32 @@ import minimist from 'minimist';
 import posthtml from 'posthtml';
 import transformModules from './modules/modules-transform';
 import transformScriptPaths from './scripts/scripts-transform';
-import transformStories from './stories/stories-transform';
 import transformCss from './css/css-transform';
 
 const argv = minimist(process.argv.slice(2));
-const transforms = [transformStories(), transformScriptPaths(), transformCss()];
+const FORTESTING = argv._.includes('integration');
```

Comment:
```suggestion
const FOR_TESTING = argv._.includes('integration');
```

Reply:
so the reason why i use it without the underscore actually is to mimic the `--fortesting` flag we have. wdyt?